unlock-app/src/components/content/RecoverContent.tsx

```diff
@@ -48,7 +47,7 @@ export const RestoreAccount = ({
 
           // We need to log the user in from the recoveryPhrase + recoveryKey!
           // TODO: what network do we pick???
-          const unlockProvider = new UnlockProvider(config.networks[network])
+          const unlockProvider = new UnlockProvider(config.networks[1])
```

Comment:
maybe use config.preferredNetwork since we encourage that everywhere?

Reply:
Hum, what is `preferredNetwork` ?

Reply:
One we show as default in selection and other places (It's polygon currently)